config/initializers/cors.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+    allow do
+      origins 'http://localhost:3001'
+      resource '*',
+      headers: ["Authorization"],
+      expose: ["Authorization"],
+      methods: [:get, :post, :put, :patch, :delete, :options, :head],
+      max_age: 600
+    end
+  end
```

Comment:
May seem small but there will be a QA who checks this stuff later on too. The indentation is a little off here.